src/hotspot/share/jfr/support/jfrTraceIdExtension.hpp

```diff
@@ -44,11 +44,13 @@
 #define REMOVE_METHOD_ID(method) JfrTraceId::remove(method);
 #define RESTORE_ID(k) JfrTraceId::restore(k);
 
+static constexpr const uint16_t cleared_epoch_bits = 512 | 256;
```

Comment:
Q: Could the `EPOCH_CLEARED_BITS` be used instead?

Reply:
No. because I want to avoid dragging in all of the definitions in jfrTraceIdMarcros.hpp.